pypdf/filters.py

```diff
@@ -274,8 +274,7 @@ def decode(
         Args:
           data: a str sequence of hexadecimal-encoded values to be
             converted into a base-7 ASCII string
-          decode_parms: a string conversion in base-7 ASCII, where each of its values
-            v is such that 0 <= ord(v) <= 127.
+          decode_parms: unused, this filter does not use parameters.
```

Comment:
Aren't both parts stating the same?

Reply:
I am aware that the original comment does not make sense, but isn't "unused" and "this filter does not use this parameters" the same content-wise?